Library/Formula/xorg.rb

```diff
@@ -0,0 +1,470 @@
+class Xorg < Formula
+  desc "Xorg Libraries"
+  homepage "http://www.linuxfromscratch.org/blfs/view/svn/x/x7lib.html"
+  revision 7
+
+  url "http://ftp.x.org/pub/individual/lib/libFS-1.0.7.tar.bz2"
+  sha256 "2e9d4c07026a7401d4fa4ffae86e6ac7fec83f50f3268fa85f52718e479dc4f3"
+  # tag "linuxbrew"
+
+  option "with-docs", "Build additional PDF or text documentation for the libXfont"
+  option "with-doxygen", "Generate API documentation (for libxcb)"
+  option "with-check-libxcb", "Run tests (for libxcb)"
+  option "with-check-xcbproto", "Run tests (for xcb-proto)"
+
+# Xorg: required dependencies
+  depends_on "fontconfig"
+# depends_on "libxcb"      # built as a resource here
```

Comment:
Not sure exactly where to put it, but I think you're missing a dependency on `gettext`.